corehq/util/metrics/__init__.py

```diff
@@ -188,12 +188,11 @@ def metrics_gauge_task(name, fn, run_every, multiprocess_mode=MPM_ALL):
     """
     _enforce_prefix(name, 'commcare')
 
-    @periodic_task(serializer='pickle', queue='background_queue', run_every=run_every,
-                   acks_late=True, ignore_result=True)
+    @periodic_task(queue='background_queue', run_every=run_every, acks_late=True, ignore_result=True)
     @wraps(fn)
-    def inner(*args, **kwargs):
+    def inner():
         from corehq.util.metrics import metrics_gauge
-        metrics_gauge(name, fn(*args, **kwargs), multiprocess_mode=multiprocess_mode)
+        metrics_gauge(name, fn(), multiprocess_mode=multiprocess_mode)
```

Comment:
Just clarifying that the args/kwargs were unused and therefore did not need to be passed into the Celery task which would risk an exception being thrown during json serialization. Does that sound right?

Reply:
Yeah they were unused. I made the change not so much to actually fix a potential issue during execution, but just to make it much easier to read. It took me some poking around to convince myself that there was no condition under which args/kwargs would ever be used, so it was much clearer to just omit them.